### ISSUE TITLE
Test 32-bit --> 64-bit transition for next deployment

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/gcd#275]
We're migrating the **standalone** product from 32 to 64 bit. 

We need to answer the question of what happens to people who already have this installed and want to upgrade.

**Comment by joewheaton** [comment]
You need me to test it?

**Comment by MattReimer** [comment]
@joewheaton yeah I might. Let me get something online in a separate space and I'll let you know.

**Comment by MattReimer** [comment]
Ok, so here's the deal: 

Clickonce won't cross platforms at all and just dies when we try to do an upgrade. 

![screen shot 2018-05-08 at 1 36 55 pm](https://user-images.githubusercontent.com/1063391/39782717-e2a50ed8-52c7-11e8-98fb-b14658460dae.png)
![screen shot 2018-05-08 at 1 37 44 pm](https://user-images.githubusercontent.com/1063391/39782724-e490078e-52c7-11e8-895b-8cd4a75a3b01.png)

On one hand this is bad... but on another hand it's good because it refuses to continue. 

I suggest we email people we know who might be using the standalone version and tell them to uninstall and reinstall once `7.1.0` is out.

Regrettable but this should be a one-time workaround.

**Comment by joewheaton** [comment]
This is fine. We only have a small group of folks on this, and we can do as
you suggest. No big deal. I suggest we keep this issue open (i.e. Known
Issues).

_____________________
Dr. Joseph M. Wheaton
435-554-1247 (direct) | http://www.joewheaton.org
